tests/tests_pytorch/loggers/test_mlflow.py

```diff
@@ -277,3 +278,45 @@ def test_mlflow_logger_finalize_when_exception(*_):
     assert logger._initialized
     logger.finalize("failed")
     logger.experiment.set_terminated.assert_called_once_with(logger.run_id, "FAILED")
+
+
+@mock.patch("pytorch_lightning.loggers.mlflow.mlflow")
+@mock.patch("pytorch_lightning.loggers.mlflow.MlflowClient")
+@pytest.mark.parametrize("log_model", ["all", True, False])
+def test_mlflow_log_model(client, mlflow, tmpdir, log_model):
+    """Test that the logger creates the folders and files in the right place."""
+
+    # Reset mocks
+    client.return_value.log_artifact.reset_mock()
+    client.return_value.log_artifacts.reset_mock()
+    # Get model, logger, trainer and train
+    model = BoringModel()
+    logger = MLFlowLogger("test", save_dir=tmpdir, log_model=log_model)
+    checkpoint_callback = ModelCheckpoint(dirpath=tmpdir / "model_checkpoints")
```

Comment:
Is there a specific reason why you chose to create a checkpoint callback instance with this particular path? `Trainer(enable_checkpointing=True)` would also work (default) right?